actions/plugins/publish/publish/publish.sh

```diff
@@ -121,7 +121,7 @@ sha=$(git rev-parse HEAD)
 popd > /dev/null
 
 # Publish the plugin
-echo "Publishing to $gcom_api_url..."
+echo "Publishing to $gcom_api_url"
```

Comment:
Removing the three dots as they add confusion in logs, you might think the whole url is cut out